Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM yastdevel/cpp
+RUN zypper --gpg-auto-import-keys --non-interactive in --no-recommends \
+  openslp-devel \
+  yast2 \
+  yast2-ruby-bindings
```

Comment:
No, I just want to avoid installing these specific packages for all the other YaST modules.